protobuf/definition/service.proto

```diff
@@ -9,6 +9,7 @@ message Service {
   string sid = 12;                          // Service's sid.
   string name = 1;                          // Service's name.
   string description = 2;                   // Service's description.
+  Configuration configuration = 8;          // Configurations related to the service
```

Comment:
This is not related to this PR - it should be done in different PR.

Reply:
that's the case, look at the description of the PR, depending on #860